trello/features/functional/BoardFunctional.feature

```diff
@@ -50,7 +50,7 @@ Feature: Board
       | id  | organization_test.id |
```

Comment:
Is it correct this id value in this line code? maybe this id can go joined with url value, so this is step is to delete
check please

Reply:
this part is Enrique's code